INN/umbrella-mwcir
# Homepage updates

## Changes

This pull request makes the following changes:

- ~Bumps Largo to 0.6.4~ removed, see https://github.com/INN/umbrella-mwcir/issues/32#issuecomment-587596894
- Copies largo's `partials/sticky-posts.php` to this repo, removes the related posts section, adds a top term, and embiggens the image
- Makes the Homepage Bottom widget area encompass one single column, rather than alternating left/right columns.
- Adds homepage-specific styles to space out the widgets in the homepage bottom area and add borders between them.

![Screenshot_2020-02-17 Investigate Midwest - The Midwest Center for Investigative Reporting 2](https://user-images.githubusercontent.com/1754187/74685511-c7611d80-519c-11ea-8120-1d8af1b8a41e.png)
![Screenshot_2020-02-17 Investigate Midwest - The Midwest Center for Investigative Reporting](https://user-images.githubusercontent.com/1754187/74685514-ca5c0e00-519c-11ea-8d3b-e9fa13af1e08.png)


## Why



Homepage changes for #27
Largo update for #32

## Testing/Questions

Features that this PR affects:

- Homepage


Questions that need to be answered before merging:

- [x] Should the text in these widgets not wrap around the images?
- [x] Should this PR also implement the before-footer section from the wireframe?

<img width="903" alt="Screen Shot 2020-02-17 at 17 08 22" src="https://user-images.githubusercontent.com/1754187/74689186-3a23c600-51a8-11ea-876f-39fb251a4300.png">


Steps to test this PR:

1. Check out this update
2. From the root of the repo, `git submodule update --init --recursive` to update Largo
3. In Appearance > Theme Options > Layout, (`/wp-admin/themes.php?page=options-framework` if you're affected by https://github.com/INN/largo/issues/1573), set:
    - Home Template to "Blog"
    - Sticky Posts to show
    - Homepage Bottom to widgets
4. In Appearance > Widgets, put a number of Recent Posts widgets there.
    - titles set to category names
    - draw from various categories
    - 3 posts
    - 140px thumbnails, left
    - yes byline
    - hide date
5. View the homepage at various screen sizes.
